### ISSUE TITLE
Temperature Calibration Tower Missing

### ISSUE THREAD

**Issue by 67RS427** [QIDITECH/QIDIStudio#32]
macOS Sonoma v14.7
Qidi Studio v1.9.7.53

Right Click QidiStudio.app, Show package contents - /Applications/3D PRINTING/QIDIStudio.app/Contents/resources/calib/temperature_tower/temperature_tower.stl

The temp tower STL is shown, but not available in the app. The other calibration options should be shown as well.

**Comment by QIDITECH** [comment]
#24
Orca Calibrations have many bugs, and it is easy for beginners to misoperate.
We will choose the calibration method suitable for our printer after testing, and make a detailed introduction and guidance.

**Comment by 67RS427** [comment]
I understand but a temperature tower calibration is very important. Please consider fixing the STL and adding it to your calibration menu. Thanks!

**Comment by dssamusaran** [comment]
I do second this idea.

Even if it's not perfect in Orca, Temperature Calibration, as well as retraction calibration is required to tune up filaments.
Your machine works great but people need those tools to be able to use it at it's full potential with other brands of filaments.

Until now I'm tediously copying over the QidiStudio profiles to Orca just to be able to use the other calibration types there. And then tuning my QidiStudio profiles.

On the other side, I do really like the clear as water guidance and the efforts you put into the flow rate/pressure advance/volumetric speed that are available. Great work! We only need more of those. :)

**Comment by dssamusaran** [comment]
> This is honestly the worst response I've ever seen they don't give a reason as to what makes it so terrible they just say it's bad so we don't use it meanwhile every other slicer has no problem getting it working

Exactly! A temp calibration method is the first step with a new filament and at Qidi they removed it. At the pace it's going we'll never have it back.

If it helps, meanwhile I'm using this script and procedure: https://www.printables.com/model/151914-ultimate-customizable-temperature-tower

You get some stl, slice it and export the gcode. Then use the script to add the temp changes to the gcode.
You also have an openscad version that allows to do a fully customized to your need temp tower stl.

**Comment by Rudokhvist** [comment]
> Exactly! A temp calibration method is the first step with a new filament and at Qidi they removed it. 

They didn't remove it. QIDIStudio is based on BambuStudio (that don't have a temperature tower calibration), and not on Orca.

**Comment by dssamusaran** [comment]
> > Exactly! A temp calibration method is the first step with a new filament and at Qidi they removed it.
> 
> They didn't remove it. QIDIStudio is based on BambuStudio (that don't have a temperature tower calibration), and not on Orca.

Good point. I always thought it was based on orca rather than Bambu studio.

Still, both of them should implement a temp tower or any other system to calibrate the temperature if they feel there is a better way.

**Comment by bgswiss1** [comment]
> > Exactly! A temp calibration method is the first step with a new filament and at Qidi they removed it.
> 
> They didn't remove it. QIDIStudio is based on BambuStudio (that don't have a temperature tower calibration), and not on Orca.

I looked it up and you're right but that makes it slightly worse because when looking it up it said orca slicer is based off of bambu. orca slicer has so many calibrations that it makes this an even worse argument.(if another slicer using the same base code as you is able to carry the entire calibration suite why does qidi only give us three)

**Comment by Hammerfest** [comment]
@QIDITECH 
Please restore/readd temperature calibration.

Bunch of failed prints today on my Max4 all because you could not be bothered to include basic calibration and none of your included profile were correct for my ASA (15c off optimal printing temps).

Hopefully you will also update QIDIStudio to include the filaments you have on the MAX4 UI, love trying to Sync the QiDi box and it tell me the filament on the printer does not exist in the slicer, so it just defaults to PLA...
